bin/desi_tile_vi

```diff
@@ -44,10 +44,10 @@ def parse(options=None):
                         help = 'your name or initials')
     parser.add_argument('--viewer', type = str, default = "eog", required=False,
                         help = 'image viewer (default is eog)')
-    parser.add_argument('--new', action = 'store_true', required=False,
-                        help = 'only inspect new tiles (equivalent to --qastatus none)')
-    parser.add_argument('--qastatus', type=str, default=None, required=False,
-                        help = 'only inspect tiles with this QA status (e.g. "unsure")')
+    parser.add_argument('--qastatus', type=str, default='none', required=False,
+                        help = ('only inspect tiles with this QA status '
+                                '(e.g. "unsure").  Default of none shows only new '
+                                'tiles.'))
```

Comment:
It took me a minute to parse that "none" was an actual QA state, rather than a stand-in for the later test `if args.qastatus is not None` which will always be True since the default is the string 'none' (which is not None...).  Please add to this docstring all the standard states that could be selected.  It is good that you describe that the default "none" shows only new tiles, since that isn't obvious from "none" alone.